docker/src/main/java/clocker/docker/entity/container/DockerContainerImpl.java

```diff
@@ -420,13 +425,20 @@ private DockerTemplateOptions getDockerTemplateOptions() {
         }
         options.env(env);
 
-        // Entrypoint
+        // Entrypoint and commands
+        // TODO parse string into list?
```

Comment:
See Brooklyn's `TypeCoercions`, which does `JavaStringEscapes.unwrapJsonishListIfPossible(input)`. Would that be useful? But probably only worth adding something like that if really is needed.